server.js

```diff
@@ -0,0 +1,104 @@
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config()
+}
+
+const express = require('express')
+const app = express()
+const bcrypt = require('bcrypt')
+const passport = require('passport')
+const flash = require('express-flash')
+const session = require('express-session')
+const methodOverride = require('method-override')
+const server = require('http').createServer(app);
+const WebSocket = require('ws');
+const initializePassport = require('./passport-config')
+initializePassport(
+  passport,
+  email => users.find(user => user.email === email),
+  id => users.find(user => user.id === id)
+)
+
+const users = []
+
+const wss = new WebSocket.Server({server});
+
+app.set('view-engine', 'ejs')
+app.use(express.urlencoded({ extended: false }))
+app.use(flash())
+app.use(session({
+  secret: process.env.SESSION_SECRET,
+  resave: true,
+  saveUninitialized: true
+}))
+app.use(passport.initialize())
+app.use(passport.session())
+app.use(methodOverride('_method'))
+
+wss.on('connection', function connection(ws){
+  console.log('A new client Connection!')
+  ws.send('Welcome new Client');
+
+  ws.on('message', function incoming(message){
+    console.log('received: %s', message);
+    ws.send('Got your message its :' + message)
+  });
+});
```

Comment:
The WebSocket implementation needs to be updated to stream messages from Kafka consumer back to the client that initiated the request.
A sample can be found here: https://medium.com/@stressed83/live-streaming-data-using-kafka-node-js-websocket-and-chart-js-8750acad549c

Additional reference:
https://www.confluent.io/events/kafka-summit-europe-2021/delivering-from-kafka-to-websockets/
https://dev.to/victorgil/kafka-websockets-angular-event-driven-microservices-all-the-way-to-the-frontend-12aa